bin/luatest

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env tarantool
+print(require("tarantool").version)
```

Comment:
1. Could you add a blank line between the shebang and print?
2. Could you please use `'` instead of `"`?
3. I think we need to highlight this somehow. For example, 

```lua
print(('Tarntool version is %s'):format(require('tarantool').version))
```

Reply:
All is done.